tests/functions/test_sort_naturally.py

```diff
@@ -0,0 +1,30 @@
+"""
+Tests for sort_naturally.
+
+Some places where this test suite could be improved:
+
+- Replace example-based test
+with Hypothesis-generated property-based test. [intermediate]
+- Provide another example-based test of something
+that needs to be naturally rather than lexiographically sorted.
+"""
+import pandas as pd
+import pytest
+from natsort import natsorted
+
+import janitor
+
+
+@pytest.fixture
+def well_dataframe():
+    data = {
+        "Well": ["A21", "A3", "A21", "B2", "B51", "B12"],
+        "Value": [1, 2, 13, 3, 4, 7],
+    }
+    df = pd.DataFrame(data)
+    return df
+
+
+def test_sort_naturally(well_dataframe):
+    sorted_df = well_dataframe.sort_naturally("Well")
+    assert sorted_df["Well"].tolist() == natsorted(well_dataframe["Well"])
```

Comment:
Could also prob test to make sure the data remains uncorrupted (by, for example, numerically sorting the new and old `Value` columns and then comparing them for equality)

Reply:
I'm going to work on this one tomorrow morning. My head is pounding from a lack of coffee this evening. (Been sluggish all day.)

Reply:
OK, just got this going.